cli/config.go

```diff
@@ -65,18 +115,29 @@ func loadConfig(cmd *cobra.Command) (Config, error) {
 	} else {
 		opts = append(opts,
 			config.WithPath("./"),
-			config.WithName("compass"),
+			config.WithName("compass.yaml"),
```

Comment:
@sudo-suhas laoding the options `WithName("compass")` gives an error `error in reading config: yaml` when the conifg file `compass.yaml` is missing in the root folder.

Instead it should be giving an error, `file not found`. Currently it is trying to use the compass executable while trying to read the configs. 

I also tried to add the `WithType("yaml")` but it still gives the same error. Can we rename the `compass.yaml` to something like `.conifg.yaml`. 

I need the correct error to be handled for searching for file made from the init command, which will be called only when the config file is missing in the root.

The extention "yaml" in line 118 will be discarded once we go ahead with something which throws the right error.

Reply:
I am not clear on why we care about the exact error message being returned. We were anyway proceeding with defaults when the config file was not found.